client/src/main/proto/spine/base/command.proto

```diff
@@ -72,20 +70,8 @@ message CommandContext {
     // The ID of the command.
     CommandId command_id = 1 [(required) = true];
 
-    // The ID of a tenant in a multi-tenant application.
-    //
-    // This field is not populated in single-tenant applications.
-    //
-    users.TenantId tenant_id = 2;
-
-    // The user who created the command.
-    users.UserId actor = 3 [(required) = true];
-
-    // The time when the command was created.
-    google.protobuf.Timestamp timestamp = 4 [(required) = true];
-
-    // The zone offset from which the command is made.
-    time.ZoneOffset zone_offset = 5 [(required) = true];
+    // Information about the user who created the command.
```

Comment:
As long as the `ActorContext` has timestamp and zone offset, that's not all `Information about the user`.